ansible/swap_syncd.yml

```diff
@@ -47,14 +47,14 @@
       become: true
       sysctl:
         name: "net.core.rmem_max"
-        value: 509430500
+        value: 609430500
```

Comment:
It is always better to keep the PRs separate. Just in case we want to revert a PR, it will be easy then.